marshmallow/utils.py

```diff
@@ -334,16 +333,34 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
-    """Given a callable, return a tuple of argument names. Handles
+def get_func_args2(func):
+    """Given a callable, return a list of argument names. Handles
```

Comment:
@sabinem You need to take your new commits and put them on top of new branch. This is done with
```
git rebase <old_base> <your_branch> --onto <new_base>
```
e.g.
```
git rebase upstream/dev issue_540_annotations_bug --onto upstream/issue540
```
It really helps to understand how Git works.

@lafrech Best practice is to rebase: it allows fixing all merge conflicts so that CI/Github can do merges for you, simplifies per-commit reviewing and makes history nice.